core/Pos/Core/Arbitrary.hs

```diff
@@ -62,6 +62,8 @@ instance Arbitrary Types.Address where
         Types.UnknownAddressType <$> choose (3, 255) <*> scale (min 150) arbitrary
         ]
 
+-- NOTE: Deriving an 'Arbitrary' instance for wrappers over integral types does not use
```

Comment:
I wrote this and the next comment as a reminder of this information, but I quickly realized they're not worded very well because I'm not specific and detailed enough for them to make sense to anyone that drops into the codebase and takes a look. I'll write them on my Slack DMs to myself, this probably shouldn't be in the codebase.